js/views/pageNavVw.js

```diff
@@ -136,10 +137,32 @@ module.exports = Backbone.View.extend({
     return this;
   },
 
+  navNotificationsClick: function(e){
+    "use strict";
+    e.stopPropagation();
+    var targ = this.$el.find('.js-navNotificationsMenu');
+    targ.siblings('.popMenu').addClass('hide');
+    if(targ.hasClass('hide')){
+      targ.removeClass('hide');
+      $('#overlay').removeClass('fadeOut hide');
```

Comment:
Adding the fadeOut and hide classes together will just apply the hide class, since it immediately sets display:none. Just use one of them.